cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -222,9 +222,9 @@ public void execute() {
             //// check if module name given is not absolute.
             if (Paths.get(argList.get(0)).isAbsolute()) {
```

Comment:
Not related to this PR, but what about if the user gives a relative path? @hevayo @hemikak

Reply:
Got to know that it will not work for a relative path.